js/accordion.js

```diff
@@ -0,0 +1,21 @@
+(function ($) {
+
+	'use strict';
+
```

Comment:
When reusing an element in jQuery, the selectors should be cached when possible so that jQuery doesn't have to seek through the DOM for the element another time. This can increase performance quite a bit when a script uses the same selector many times.

`var js_accordion = $( ".js-accordion" ); // and then use js_accordion wherever`

I'll often start the variable with a `$` so that I know it's a cached selector:

`var $js_accordion = $( ".js-accordion" );`